Breaking: make 'useJSXTextNode:true' by default (#544)
eslint/typescript-eslint-parser: README.md

**`jsx`** - default `false`. Enable parsing JSX when `true`. More details can be found [here](https://www.typescriptlang.org/docs/handbook/jsx.html).

> **`useJSXTextNode`** - default `true`. The JSX AST changed the node type for string literals inside a JSX Element from `Literal` to `JSXText`. When value is `true`, these nodes will be parsed as type `JSXText`. When value is `false`, these nodes will be parsed as type `Literal`. Please set `false` if you use this parser on ESLint v4.

Reviewer: I'm wondering if the note at the end of this comment should be made more prominent in some way (e.g., apply formatting, or put it on its own line, or...)

Maybe someone else has a thought on if this should be done?

Author: How about...?

```suggestion
**`useJSXTextNode`** - default `true`. Please set `false` if you use this parser on ESLint v4. If this is `false`, the parser creates the AST of JSX texts as the legacy style.
```

Reviewer: That could work, thanks!

Author: Thank you.

I used the "suggested change β" feature for the first time :)
